src/native/fs/fs_napi.cpp

```diff
@@ -34,6 +35,8 @@ namespace noobaa
 
 DBG_INIT(0);
 
+int (*gpfs_fcntl)(gpfs_file_t fileDesc, void* fcntlArgP);
```

Comment:
We probably want to declare this pointer static (file scoped), also better choose a name that doesnt exist in the gpfs header to avoid confusion, like dlsym_gpfs_fnctl.
```suggestion
static int (*dlsym_gpfs_fcntl)(gpfs_file_t file, void* arg);
```

Reply:
done